### PR TITLE
281 end of page nav links

### PR DESCRIPTION
**Pull request by Nikomus**
Trello card: https://trello.com/c/gKwdif95/281-end-of-page-nav-links

## Description

- Add some preliminary styling to the links

image

@neontribe/sea-map

### CODE REVIEW

File: src/Copy/Text.elm [hunk 1]
```diff
@@ -313,53 +316,53 @@ t key =
             "[cCc] Help in times of COVID"
 
         HelpSelfCategory1Resource1Title ->
-            "[cCc] Economic abuse and the coronavirus outbreak"
```
Comment (katjam) [inline src/Copy/Text.elm]: I think these need to be the examples I put in to help Stuart and Georgia see in context.

File: src/Copy/Text.elm [hunk 2]
```diff
@@ -167,8 +167,11 @@ t key =
         Journey6Age ->
             "61"
 
+        ToDefinitionReassuringText ->
+            "If this sounds familiar know that you’re not alone. "
```
Comment (katjam) [inline src/Copy/Text.elm]: Is the trailing space intentional?

File: src/Page/GetHelp.elm [hunk 3]
```diff
@@ -21,7 +20,14 @@ view =
             , card (t GetHelpSection2Title) (t GetHelpSection2Quote) (t GetHelpSection2Description) CallSupport
             , card (t GetHelpSection3Title) (t GetHelpSection3Quote) (t GetHelpSection3Description) SeeOrgs
             ]
-        , a [ href (t HelpSelfGridPageSlug) ] [ text (t ToHelpSelfFromGetHelpLink) ]
+        , verticalSpacing
+        , nav [ css [ navListStyle ] ]
+            [ p [ css [ reassuringStyle ] ]
+                [ text (t ToHelpSelfReassuringText) ]
+            , a
+                [ href (t HelpSelfGridPageSlug), css (navLinkStyle :: navItemStyles) ]
```
Comment (katjam) [inline src/Page/GetHelp.elm:28]: Nice!

File: src/Copy/Text.elm (hunk 2 above)
Comment (Nikomus) [reply src/Copy/Text.elm]: Nope - fixed!

**Review by katjam — APPROVED** [review]
This looks good to go now. :kite:

**Pull request merged**